clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -142,6 +142,13 @@ copies of the binaries and reading the raw bytes for each instruction
 in order to obtain the opcode and full operand information.
 See also \ref sec_drcachesim_core.
 
+drmemtrace analysis tools may use the
+#dynamorio::drmemtrace::decode_cache_t library, which handles the
+heavy-lifting of decoding the trace instructions using either the embedded
+encodings in the trace or the encodings from the app binaries, and managing
+their cache (including invalidating stale encodings based on the
```

Comment:
nit: "managing their cache" w/o prior reference to a cache sounds off to me: how about "and manages caching the encodings" or sthg.

Reply:
changed to "and manages their decode info"